docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -0,0 +1,320 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _ifci_px4_flight_controller:
+
+****************************************************
+Setting up PX4 Firmware for Use with IFCI
+****************************************************
+.. Converted from Jordan's google docs tutorial
+This tutorial covers how to build and set up `Vertiq’s PX4 fork <https://github.com/iq-motion-control/PX4-Autopilot>`_ which provides access to Vertiq’s IQUART protocol. With IQUART integrated into your flight controller, you gain the ability to control, configure, and receive telemetry from all connected modules through a single serial port. Please note that in order to control your module with our PX4 fork, your module must support the :ref:`IQUART Flight Controller Interface (IFCI)<controlling_ifci>`. The features supported by your module and firmware style can be found on your module’s family page.
+
+Setting Up the PX4 Toolchain
+=============================
+
+In order to use our PX4 fork, you must install the PX4 toolchain. We recommend that you follow `PX4's guides <https://docs.px4.io/main/en/dev_setup/dev_env.html>`_ in order to install the toolchain for your specific device. Note that when the PX4 guide gives the instruction to run ``git clone https://github.com/PX4/PX4-Autopilot.git --recursive`` it should be replaced with ``git clone https://github.com/iq-motion-control/PX4-Autopilot --recursive`` in order to clone Vertiq's fork rather than the mainline PX4.
+
+.. warning::
+    If you do not replace the standard PX4 fork with our fork, none of the future steps will be possible. Make sure you properly replace ``git clone https://github.com/PX4/PX4-Autopilot.git --recursive`` with ``git clone https://github.com/iq-motion-control/PX4-Autopilot --recursive`` as mentioned above.
+    
+Once the toolchain is installed navigate into the top level PX4-Autopilot directory and add the remote original PX4 remote repository:
+
+.. code-block:: bash
+
+    git remote add upstream git@github.com:PX4/PX4-Autopilot.git
+    git fetch upstream
+
+This should result in tags being added to the local repository. Once this is done you should be able to build PX4. These steps are from the `PX4 Git documentation <https://docs.px4.io/main/en/contribute/git_examples.html>`_. If the tags are not added then the compilation will fail.
+
+.. once we are in px4, this whole section can be replaced with install px4 toolchain as described by them
+
+Setting Up PX4 for IQUART and Building
+======================================
+
+Once the toolchain is set up, you must change the settings for your board to turn on Vertiq IQUART integrations. To do this, enter your PX4 directory and use the command below, but replace ``<your-flight-control-board>`` with your flight control board's name. Additionally a command with the example board we are using is shown below for reference.
```

Comment:
Take out the, "Additionally a command..." sentence. Just before the second code block say, "for this example, we are building for <whatever flight controller this is>. So, we run <command example"